components/datetime/benches/fixtures/tests/lengths.json

```diff
@@ -87,7 +69,7 @@
                 "options": {
                     "length": {
                         "date": "long",
-                        "time": "long"
+                        "time": "short"
```

Comment:
**Author Clarification**

`DateTimeFormat` no longer supports patterns that contain time-zone symbols, and returns an `Err` on construction. This means it can no longer be used with `Time::Long` and `Time::Full`. 

These are tested by `ZonedDateTimeFormat` which pulls from `lengths_with_zones.json`.